ansible/ansible.cfg

```diff
@@ -117,16 +117,20 @@ ansible_managed = Ansible managed: {file} modified on %Y-%m-%d %H:%M:%S by {uid}
 
 # set plugin path directories here, separate with colons
 action_plugins     = /usr/share/ansible_plugins/action_plugins
-callback_plugins   = /usr/share/ansible_plugins/callback_plugins
+callback_plugins   = /opt/env/atmo/lib/python2.7/site-packages/subspace/plugins/callback:/opt/env/atmo/local/lib/python2.7/site-packages/ansible/plugins/callback
```

Comment:
Note to self: Will need a better long-term solution, since we probably don't want to hard-code dependency to virtualenv (or log path)